scripts/jenkins/gcc-dynamic.groovy

```diff
@@ -3,27 +3,45 @@ def defaultCMakeOptions =
     '-DOGS_LIB_BOOST=System ' +
     '-DOGS_LIB_VTK=System ' +
     '-DBUILD_SHARED_LIBS=ON ' +
-    '-DOGS_BUILD_UTILS=ON'
+    '-DOGS_BUILD_UTILS=ON '
 
 def configure = new ogs.configure()
 def build = new ogs.build()
 def post = new ogs.post()
 def helper = new ogs.helper()
 
-stage('Configure (Linux-Docker-Dynamic)') {
+stage('Configure (envinf1)') {
     configure.linux(cmakeOptions: defaultCMakeOptions, env: 'envinf1/cli.sh', script: this)
+    configure.linux(cmakeOptions: defaultCMakeOptions + '-DOGS_USE_MPI=ON',
```

Comment:
As far as I knew that there is no stand alone implementation and test for  option of OGS_USE_MPI. OGS_USE_MPI should be on only when OGS_USE_PETSC is ON.